js/src/ui/Page/page.css

```diff
@@ -18,7 +18,7 @@
 .layout {
   padding: 0.25em 0.25em 1em 0.25em;
 
-  > * {
-    margin-bottom: 0.75em;
+  &>div {
```

Comment:
Any reason for that ? The `&` is useless, and we might not want to limit to `div`s. `padding` instead of `margin` is what was breaking the `event` pannel in Contracts